plugins/lsp/language-servers/efmls-configs.nix

```diff
@@ -5,147 +5,31 @@
-      php
-      prettierd
-      prettypst
-      proselint
-      protolint
-      pylint
-      rubocop
-      ruff
-      rustfmt
-      scalafmt
-      selene
-      shellcheck
-      shellharden
-      shfmt
-      smlfmt
-      sqlfluff
-      statix
-      stylua
-      taplo
-      typstfmt
-      typstyle
-      uncrustify
-      vale
-      yamllint
-      yapf
-      ;
-    inherit (python3.pkgs)
-      autopep8
-      flake8
-      mdformat
-      vulture
-      ;
-    inherit (nodePackages)
-      eslint
-      eslint_d
-      prettier
-      alex
-      sql-formatter
-      stylelint
-      textlint
-      write-good
-      ;
-    inherit (phpPackages) phan phpstan psalm;
-    inherit (luaPackages) luacheck;
-    inherit (haskellPackages) fourmolu;
-    ansible_lint = ansible-lint;
-    chktex = texliveMedium;
-    clang_format = clang-tools;
-    clang_tidy = clang-tools;
-    clj_kondo = clj-kondo;
-    cmake_lint = cmake-format;
-    dartfmt = dart;
-    dotnet_format = dotnet-runtime;
-    fish_indent = fish;
-    gofmt = go;
-    goimports = go-tools;
-    golangci_lint = golangci-lint;
-    google_java_format = google-java-format;
-    go_revive = revive;
-    latexindent = texliveMedium;
-    lua_format = luaformatter;
-    markdownlint = markdownlint-cli;
-    mcs = mono;
-    nixfmt = nixfmt-classic;
-    phpcbf = phpPackages.php-codesniffer;
-    php_cs_fixer = phpPackages.php-cs-fixer;
-    phpcs = phpPackages.php-codesniffer;
-    prettier_d = prettierd;
-    slither = slither-analyzer;
-    staticcheck = go-tools;
-    terraform_fmt = terraform;
-    vint = vim-vint;
-    write_good = write-good;
-    yq = yq-go;
-  };
   # Filter packages that are not compatible with the current platform
-  toolPkgs = filterAttrs (a: pkg: meta.availableOn pkgs.stdenv.hostPlatform pkg) allToolPkgs;
+  toolPkgs = lib.filterAttrs (
+    a: pkg: lib.meta.availableOn pkgs.stdenv.hostPlatform pkg
+  ) packagingStatus.packaged;
 in
 {
   options.plugins.efmls-configs = {
-    enable = mkEnableOption "efmls-configs, premade configurations for efm-langserver";
+    enable = lib.mkEnableOption "efmls-configs, premade configurations for efm-langserver";
 
     package = helpers.mkPluginPackageOption "efmls-configs-nvim" pkgs.vimPlugins.efmls-configs-nvim;
 
-    externallyManagedPackages = mkOption {
-      type = types.either (types.enum [ "all" ]) (types.listOf types.str);
+    externallyManagedPackages = lib.mkOption {
+      type = with lib.types; either (enum [ "all" ]) (listOf str);
       description = ''
         Linters/Formatters to skip installing with nixvim. Set to `all` to install no packages
       '';
       default = [ ];
     };
 
-    toolPackages = attrsets.mapAttrs (
-      tool: pkg:
-      mkOption {
-        type = types.package;
-        default = pkg;
-        description = "Package for ${tool}";
-      }
+    toolPackages = lib.attrsets.mapAttrs (
+      tool: pkg: helpers.mkPluginPackageOption "${tool}" pkg
```

Comment:
My bad, I think we want `helpers.mkPackageOption` instead, since this isn't the _plugin's_ package.

Also `"${tools}"` can just be `tools`, right?

Reply:
That's hasty refactorings for you....